docs/colorpicker.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Hyper-Link DescribedBy JSON Editor Example</title>
+  <meta http-equiv="content-type" content="text/html; charset=utf-8">
+
+  <!-- Enable responsive viewport -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <!-- jQuery -->
+  <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+
+  <!-- Bootstrap3 -->
+  <!-- Latest compiled and minified CSS -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <!-- Optional theme -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" >
+  <!-- Latest compiled and minified JavaScript -->
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" ></script>
+
+  <!-- JSON-Editor -->
+  <script src="../dist/dev/jsoneditor.js"></script>
+  <!-- JSON-Editor -->
+  <script src="https://unpkg.com/vanilla-picker@2"></script>
+  <style type="text/css">
+  body {
+    margin: 1em;
+  }
+
+  </style>
+  </head>
+  <body>
+    <h1>Hyper-Link DescribedBy JSON Editor Example</h1>
+
+    <div id='editor_holder'></div>
+    <button id='submit'>Submit (console.log)</button>
+
+    <script>
+
+      // Initialize the editor
+      var editor = new JSONEditor(document.getElementById('editor_holder'), {
+        theme: 'bootstrap3',
```

Comment:
We should probably not use Bootstrap4 theme in examples as this theme will be removed in next? release.